requirements/edx/development.txt

```diff
@@ -42,7 +42,7 @@ git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.1.6#egg=xblock-d
 -e git+https://github.com/edx-solutions/xblock-google-drive.git@138e6fa0bf3a2013e904a085b9fed77dab7f3f21#egg=xblock-google-drive
 git+https://github.com/open-craft/xblock-poll@add89e14558c30f3c8dc7431e5cd6536fff6d941#egg=xblock-poll==1.5.1
 -e git+https://github.com/raccoongang/xblock-pdf.git@sgab-v.1.0.1#egg=xblock-pdf
-git+ssh://git@github.com/edly-io/edly-panel-edx-app.git@develop-multisite#egg=edly-panel-app
+git+ssh://git@github.com/edly-io/edly-panel-edx-app.git@3.0.4#egg=edly-panel-app==3.0.4
```

Comment:
why do we need this change?

Reply:
It is already being used in `development.txt` already. I think it is used when devstack is setup.

Reply:
Yes. but we need it to install the `develop-multisite` branch for this repo, not a release tag

Reply:
Actaully I'm utilizing requirements cache feature of circleci. If we use `develop-multisite` branch which means we don't have to change `edx/requirements/testing.txt` and as result of it circleci would use cached requirements which can break units tests.